kubernetes/samples/charts/voyager/README.md

```diff
@@ -1,98 +1,120 @@
 # Install and configure Voyager
 
 ## A step-by-step guide to install the Voyager operator
-AppsCode has provided a Helm chart to install Voyager. See the official installation document at https://appscode.com/products/voyager/7.4.0/setup/install/.
+AppsCode has provided a Helm chart and instructions to install Voyager. See the official installation document at:
+- https://appscode.com/products/voyager/latest/setup/install/
 
-As a demonstration, the following are the detailed steps to install the Voyager operator by using a Helm chart on a Linux OS.
+Also check the Kubernetes version support matrix based on your Kubernetes installation at:
+- https://github.com/appscode/voyager#supported-versions
+
+As a *demonstration*, the following are steps to install the Voyager operator by using Helm 2 on a Linux OS.
 
 ### 1. Add the AppsCode chart repository
 ```
 $ helm repo add appscode https://charts.appscode.com/stable/
 $ helm repo update
 ```
 Verify that the chart repository has been added.
+
+Using Helm 3:
```

Comment:
This PR is pointed toward the master branch, but we don't yet support Helm 3 there yet.  How about we target this PR toward develop?  We are planning on releasing a 2.5.0 release later this week.

Reply:
This is about using Helm 3 with the voyaer charts, this was added because Monica asked for it...

I already explained the reason for `master`. do you want to rebase against `develop`? Pani is making changes so it might be easier to wait for his change to be merged?